### DISCUSSION TITLE
zh/guide/drivers/quqi

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#328] [Announcements]
# zh/guide/drivers/quqi

曲奇云盘官网：https://quqi.com 手机号、密码 登录手机号和密码，建议使用 Cookie (#cookie)方式登录 Cookie 使用Cookie登录 可保持多终端同时在线使用，如果使用 手机号、密码进行挂载会顶掉其它终端登录状态 获取方式：随意在F12网络请求中找到一个携带Cookie参数的进行复制即可; cookie 根文件夹ID ...

https://alist.nn.ci/zh/guide/drivers/quqi.html

**Comment by ipme** [comment]
添加 存储的时候 没有 曲奇网盘  的选项啊

**Reply by esyka114514** [reply]
同

**Reply by hcymc** [reply]
同

**Comment by 16175186** [comment]
请更新到v3.31.0
Add quqi driver (#5899 close #5251)  -  by @EchoResponse and @xhofe in #5899 and #5251 (442c2)

**Comment by foxdodo** [comment]
下载限速100kb/s

**Reply by vpei** [reply]
文件大小限制1G以内。

**Reply by foxdodo** [reply]
玩个🔨😆

**Reply by EchoResponse** [reply]
cdn接口多连接速度会好一些

**Comment by eying99** [comment]
曲奇网盘的群组云，怎么连接？
目前只能连接私人云。

**Reply by EchoResponse** [reply]
目前只支持私人云，后面有时间我把群组云加上

**Comment by Ender-SaMaQWQ** [comment]
导入/移动 曲奇云盘文件提示 
>目标文件夹访问被拒绝
>你需要权限来操作  

是咋回事

**Reply by Ender-SaMaQWQ** [reply]
补充一下
用的Alist版本3.35.0
挂载到本地用的软件是RaiDrive2023.9.35

**Reply by 816633** [reply]
同

**Reply by eclipsetb** [reply]
同

**Comment by eclipsetb** [comment]
曲奇云盘用RaiDrive挂载到本地，没有写的权限，直接拖入文件，上传不了曲奇云盘!

**Comment by sunhaoming665** [comment]
**曲奇云盘即将停止运行，无法上传等，转移文件请尽快！！！**
